### DISCUSSION TITLE
Multi Tenancy

### DISCUSSION THREAD

**Discussion by vanvanni** [Cockpit-HQ/Cockpit#60] [General]
Hi,

I was wondering if Cockpit supports multi tenancy or it has extensibility that would make this possible in any way?

**Comment by gbrlmza** [comment]
Spaces are doing the job for me: https://getcockpit.com/documentation/core/concepts/spaces

**Reply by vanvanni** [reply]
Sounds great, Ill have a look! Another question: can I extend cockpit myself? Or are there only set addons from them when using premium?

**Reply by aheinze** [reply]
Sure, you can create your own addons and extend Cockpit.

**Reply by vanvanni** [reply]
> Sure, you can create your own addons and extend Cockpit.

Great! Are there any docs on this? I only found about the pro addons :$
